examples/machine-learning/README.md

```diff
@@ -94,72 +82,103 @@ gcloud storage buckets create gs://${BUCKET} --project=${PROJECT_ID} \
 gcloud storage buckets update gs://${BUCKET} --versioning
 ```
 
+Modify all 3 blueprints to configure the new bucket to serve as a Terraform
+remote backend:
+
+```yaml
+terraform_backend_defaults:
+  type: gcs
+  configuration:
+    bucket: customer-bucket # modify to bucket created above
+```
+
+### Set default values
+
+Modify the the blueprint deployment variables `project_id`, `region`, `zone`, in
+the `vars` block of all 3 blueprints:
+
+```yaml
+  project_id: customer-project
+  region: customer-region
+  zone: customer-zone
+```
+
+### Set kernel-patched OS image
+
+Obtain values for `source_image_project_id` and `source_image` from your Google
+Cloud representative. Set them at approximately lines 33 and 34 of
+`ml-slurm-a3-1-image.yaml`. 
+
+```yaml
+  source_image_project_id: source-image-project-id # use value supplied by Google Cloud staff
+  source_image: source-image-name                  # use value supplied by Google Cloud staff
+```
+
+### Reservation created by Google
+
 If Google Cloud staff have created a reservation for you, then skip this step
```

Comment:
Have *not*?

Reply:
If inverting the order, then we can rephrase this: 
If Google Cloud staff have created a reservation for you, you must alter the blueprint...

Reply:
Done. I suggest reviewing the rendered copy at:

https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/4761d9a582c5cb1812beae0d4f742c353190e274/examples/machine-learning/README.md